modules/ocf/files/puppet-trigger

```diff
@@ -33,10 +33,8 @@ def trigger_run():
         sys.exit(1)
 
 
-def tail_syslog():
-    # call less in "secure" mode since this is triggered by passwordless sudo
-    os.environ['LESSSECURE'] = '1'
-    os.execvp('less', ('less', '+F', '/var/log/syslog'))
+def show_journal():
+    os.execvp('journalctl', ('journalctl', '-f', '-u', 'puppet.service'))
```

Comment:
i think you still want the `LESSSECURE=1` environment variable since journalctl invokes `less` (well, `PAGER`, which will default to `less`)